dolphinscheduler-ui/src/views/security/user-manage/components/authorize-modal.tsx

```diff
@@ -88,14 +109,66 @@ export const AuthorizeModal = defineComponent({
         confirmClassName='btn-submit'
         cancelClassName='btn-cancel'
       >
-        {type === 'authorize_project' && (
+        {/* {type === 'authorize_project' && (
           <NTransfer
             virtualScroll
             options={this.unauthorizedProjects}
             filterable
             v-model={[this.authorizedProjects, 'value']}
             class={styles.transfer}
           />
+        )} */}
+        {type === 'authorize_project' && (
+
+          <NSpace vertical>
+            <NSpace>
+              <NButton size='small' type='primary' onClick={() => this.revokeProjectByIdRequest(userId, projectIds)}>
```

Comment:
BTW, if you are using an arrow function, and you don't want to use curly brackets. It's better to add `void` after the arrow that means the arrow function return nothing.

Reply:
Suggestions received and have been revised as suggested.